src/app/index.module.ts

```diff
@@ -278,7 +278,7 @@ initModule.config(['$routeProvider', ($routeProvider: che.route.IRouteProvider)
 
 }]);
 
-const DEV = false;
```

Comment:
it should be reverted before merging, right?

Reply:
yes, it shouldn't be merged

Reply:
yep